### PR TITLE
Browser templates view should not use SWT.WEBKIT as browser style #1509

### PR DESCRIPTION
**Pull request by wimjongman**
Switched to SWT.DEFAULT to activate default browser

### CODE REVIEW

File: ui/org.eclipse.pde.ui.templates/templates_3.1/browserView/java/$className$.java [hunk 1]
```diff
@@ -54,7 +54,7 @@ public class $className$ extends ViewPart implements ISelectionListener {
 
 	@Override
 	public void createPartControl(Composite parent) {
-		fBrowser = new Browser(parent, SWT.WEBKIT);
```
Comment (wimjongman) [inline ui/org.eclipse.pde.ui.templates/templates_3.1/browserView/java/$className$.java:57]: @akurtakov should this style bit load internet explorer or does it need fixing in SWT?

Comment (merks) [reply ui/org.eclipse.pde.ui.templates/templates_3.1/browserView/java/$className$.java:57]: The default is being changed from IE to Edge during the current release cycle.

https://www.eclipse.org/lists/cross-project-issues-dev/msg19996.html

Comment (akurtakov) [reply ui/org.eclipse.pde.ui.templates/templates_3.1/browserView/java/$className$.java:57]: IMO the template should not choose any browser engine at all.

Comment (akurtakov) [reply ui/org.eclipse.pde.ui.templates/templates_3.1/browserView/java/$className$.java:57]: Yes, sorry for not giving clearer answer.

**Pull request merged**